Source/JavaScriptCore/runtime/JSObject.h

```diff
@@ -878,7 +878,7 @@ class JSObject : public JSCell {
     bool putOwnDataProperty(VM&, PropertyName, JSValue, PutPropertySlot&);
     bool putOwnDataPropertyMayBeIndex(JSGlobalObject*, PropertyName, JSValue, PutPropertySlot&);
 
-    void putOwnDataPropertyBatching(VM&, const RefPtr<UniquedStringImpl>*, const EncodedJSValue*, unsigned size);
+    void putOwnDataPropertyBatching(VM&, UniquedStringImpl**, const EncodedJSValue*, unsigned size);
```

Comment:
Should this be a `std::span<UniqueStringImpl*>`?

Reply:
It should be. But we currently skip enforcement in this file. I think I'll fix that in a follow-up, to ensure the integrity of the perf test.